easyconfigs/s/Satsuma2/Satsuma2-20220304-GCC-11.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'Satsuma2'
+local_commit = '37c5f38'
+version = '20220304'
+
+homepage = 'https://github.com/bioinfologics/satsuma2'
+description = """Satsuma2 is an optimised version of Satsuma, a tool to reliably align large and complex DNA sequences
+providing maximum sensitivity (to find all there is to find), specificity (to only find real homology) and speed (to
+accommodate the billions of base pairs in vertebrate genomes)."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/bioinfologics/satsuma2/archive/']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+patches = ['Satsuma2-%(version)s_fix-GCC-11.patch']
+checksums = [
+    '75ba38066719c189fba8630326a35afe42421437b55ee9e631973e8be115674c',  # Satsuma2-20220304.tar.gz
+    'c80755f8942f3a0c9a2598b2a1a6319f508956bc29ccdfdfb9044ec8c50cd691',  # Satsuma2-20220304_fix-GCC-11.patch
+]
+
+builddependencies = [('CMake', '3.23.1')]
+
+# Define SATSUMA2_PATH as per:
+# https://github.com/bioinfologics/satsuma2/blob/37c5f386819614cd3ce96016b423ddc4df1d86ec/README.md#running-satsuma2
+modextravars = {
+    'SATSUMA2_PATH': '%(installdir)s/bin',
+}
```

Comment:
This is the modified code when compared with the upstream easyconf

Reply:
Upstream this change and we'll install from the PR.